lib/tests/streamlit/streamlit_test.py

```diff
@@ -303,7 +303,7 @@ def test_st_help(self):
             el.doc_string.doc_string.startswith("Display text in header formatting.")
         )
         self.assertEqual(el.doc_string.type, "<class 'method'>")
-        self.assertEqual(el.doc_string.signature, "(body)")
+        self.assertEqual(el.doc_string.signature, "(body, anchor=None)")
```

Comment:
Are there any unit tests for the new behavior?

Reply:
Yes, now there are, please see [here](https://github.com/streamlit/streamlit/pull/2513/files#diff-3e8663105b92cf73c525014ad2b2b05042a42723447ef8cbf9607b0956c10da1).

Reply:
Now there are: https://github.com/streamlit/streamlit/pull/2513/files#diff-3e8663105b92cf73c525014ad2b2b05042a42723447ef8cbf9607b0956c10da1

Reply:
Great!